application/web/web_main.py

```
@@ -487,11 +487,64 @@ def map_analysis() -> Any:
-            abort(503, f"Gap analysis unavailable: {fallback_exc}")
+            db.gap_analysis(
+                neo_db=database.neo_db,
+                node_names=standards,
+                cache_key=cache_key,
+            )
+            cached = database.get_gap_analysis_result(cache_key=cache_key)
+            if cached:
+                parsed = json.loads(cached)
+                if "result" in parsed:
+                    return jsonify({"result": parsed["result"]})
+        except Exception as db_exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)